Mathlib/Analysis/Convex/Polytope.lean

```diff
@@ -0,0 +1,58 @@
+/-
+Copyright (c) 2025 Yaël Dillies, Matthew Johnson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Matthew Johnson
+-/
+import Mathlib.Analysis.Convex.Hull
+import Mathlib.Analysis.Convex.Combination
+import Mathlib.Algebra.Group.Pointwise.Finset.Basic
+
+/-!
+# Polytopes
+
+We define polytopes as convex hulls of finite sets.
+-/
+
+open scoped Pointwise
+
+variable {R 𝕜 E : Type*}
+
+section Semiring
+variable [Semiring R] [PartialOrder R] [IsOrderedRing R]
+variable [AddCommMonoid E] [Module R E] {s t : Set E} {x y : E}
```

Comment:
Should this be stated for affine spaces?

Reply:
Yes, but mathlib currently only supports convex hulls in modules